examples/nl-core-VisualAcuity-01.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-VisualAcuity-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-VisualAcuity"/>
+   </meta>
+   <extension url="http://hl7.org/fhir/StructureDefinition/observation-deviceCode">
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="419475002"/>
+            <display value="Pinhole visual acuity"/>
+         </coding>
+      </valueCodeableConcept>
+   </extension>
+   <status value="final"/>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="363983007"/>
+         <display value="visus"/>
+      </coding>
+   </code>
+   <effectiveDateTime value="2020-02-08"/>
+   <valueQuantity>
+      <value value="0.8"/>
+      <unit value="decimal"/>
+      <system value="http://unitsofmeasure.org"/>
+      <code value="{decimal}"/>
+   </valueQuantity>
+   <bodySite>
+      <extension url="http://nictiz.nl/fhir/StructureDefinition/ext-AnatomicalLocation.Laterality">
+         <valueCodeableConcept>
+            <coding>
+               <system value="http://snomed.info/sct"/>
+               <code value="7771000"/>
+               <display value="Links"/>
+            </coding>
+         </valueCodeableConcept>
+      </extension>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="81745001"/>
+         <display value="structuur van bulbus oculi (lichaamsstructuur)"/>
+      </coding>
+   </bodySite>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="16830007"/>
+            <display value="onderzoek van visus (verrichting)"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="419775003"/>
+            <display value="Best-gecorrigeerde visus"/>
+         </coding>
+      </valueCodeableConcept>
+   </component>
+   <component>
+      <code>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="421763006"/>
+            <display value="visuskaart"/>
+         </coding>
+      </code>
+      <valueCodeableConcept>
+         <coding>
+            <system value="http://snomed.info/sct"/>
+            <code value="420050001"/>
+            <display value="Ongecorrigeerde visus"/>
```

Comment:
This should be code '2' (Snellen Letterkaart) from code system 2.16.840.1.113883.2.4.3.11.60.40.4.27. This is caused by the ada instance.

Reply:
Fixed